File: WebApp/WebApp/Models/PriceList.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace WebApp.Models
{
    public class PriceList
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }

        [Required]
        [DataType(DataType.DateTime)]
        public DateTime ValidFrom { get; set; }

        [Required]
        [DataType(DataType.DateTime)]
        public DateTime ValidUntil { get; set; }

        public virtual ICollection<PriceListItem> PriceListItems { get; set; }

        public PriceList()
        {
            this.PriceListItems = new HashSet<PriceListItem>();
        }

    }
}

File: NodeJsApp/src/models/priceList.js
import { Schema, model } from 'mongoose';

const priceListSchema = new Schema(
    {
        validFrom: {
            type: Date,
            required: true
        },
        validUntil: {
            type: Date,
            required: true
        },
        priceListItems: [
            {
                type: Schema.Types.ObjectId,
                ref: 'PriceListItem'
            }
        ]
    },
    {
        versionKey: 'rowVersion'
    }
);

export default model('PriceList', priceListSchema);